feeder/__main__.py

```diff
@@ -111,4 +111,4 @@ async def shutdown_event():
 
 
 if __name__ == "__main__":
-    uvicorn.run("feeder.__main__:app", host="0.0.0.0", port=settings.http_port)
+        uvicorn.run("feeder.__main__:app", host="0.0.0.0", port=settings.http_port, ssl_keyfile=settings.mqtts_private_key, ssl_certfile = settings.mqtts_public_key)
```

Comment:
Were you able to actually use the API in your browser with these settings? I tired this when I was originally switching everything over and Chrome refused to connect to the service.

Reply:
Yes, once I added the trust of the self-signed cert.

Reply:
**Side note:** This is double indented.